src/core/modules/impl/handler/WasmHandlerApi.ts

```diff
@@ -90,6 +90,14 @@ export class WasmHandlerApi<State> extends AbstractContractHandler<State> {
     }
   }
 
+  evolveState() {
```

Comment:
is it possible to map the return type? at least sth like `EvolveState | State`?

Reply:
I don't understand the question. I added this method only to demonstrate how to use new API. Feel free to change it in whatever way you want.